nix/deps/gradle/proj.list

```diff
@@ -1,39 +0,0 @@
-app
```

Comment:
hmm this is a problem. 
seems like `make nix-update-gradle` command terminated midway..
I can push a commit and fix if you'd like @J-Son89

Reply:
That'd be great! Thanks 🙏

Reply:
Done.